realm-jni/src/io_realm_internal_SharedGroup.cpp

```diff
@@ -83,16 +97,19 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_SharedGroup_nativeCreate(
 }
 
 JNIEXPORT jlong JNICALL Java_io_realm_internal_SharedGroup_createNativeWithImplicitTransactions
-  (JNIEnv* env, jobject, jlong native_replication_ptr, jbyteArray keyArray)
+  (JNIEnv* env, jobject, jlong native_replication_ptr, jint durability, jbyteArray keyArray)
 {
     TR_ENTER()
+
+    SharedGroup::DurabilityLevel level;
+    if (!jintToDurabilityLevel(durability, level)) {
+        ThrowException(env, UnsupportedOperation, "Unsupported durability.");
+        return 0;
+    }
+
     try {
         KeyBuffer key(env, keyArray);
-#ifdef REALM_ENABLE_ENCRYPTION
```

Comment:
REALM_ENABLE_ENCRYPTION doesn't exsit in group_shared.hpp anymore. And that constructor has been removed from native header.